docs/application/native/guides/app-management/dali-widget-app.md

```diff
@@ -0,0 +1,268 @@
+
+   To manage signal connection safely, DALi provides the [Dali::ConnectionTracker](../../api/mobile/latest/classDali_1_1ConnectionTracker.html) class. A typical way to start a DALi application is to create a class derived from the `Dali::ConnectionTracker` class and use its member functions as callback functions for DALi signals.When the application receives the `InitSignal()`, it can create UI components. Connect the `WidgetApplicationExample::Create()` callback to the `Dali::WidgetApplication::InitSignal()` function:
+   ```
+    class WidgetApplicationExample : public ConnectionTracker
+    {
+      public:
+        WidgetApplicationExample( WidgetApplication& application )
+          : mApplication( application )
+        {
+          mApplication.InitSignal().Connect( this, &WidgetApplicationExample::Create );
+        }
+
+        void Create( Application& application )
+        {
+          // Register widget creating function here.
+        }
+
+      private:
+        WidgetApplication& mApplication;
+    }
+   ```
+
+3. Create own widget class
+
+   Before starting, You should know DALi background knowledge [Handle/Body Pattern: Basic Way of Using DALi Objects](handle.md).
+
+   Because of this pattern, you need to make two classes. Handle class of custom widget and Body class of custom widget class.
+
+   1. Make handle class of custom widget(sample-widget.h).
+
+   ```
+   #include <dali/public-api/adaptor-framework/widget.h>
+
+   namespace Demo
+   {
+   namespace Internal
+   {
+     class SampleWidget;
+   }
+   class SampleWidget : public Dali::Widget    // Inherit widget class
+   {
+   public:
+     SampleWidget();
+     static SampleWidget New();
+     ~SampleWidget();
+     SampleWidget( const SampleWidget& sampleWidget );
+     SampleWidget& operator=( const SampleWidget& sampleWidget );
+   public:
+     explicit SampleWidget( Internal::SampleWidget* sampleWidget );
+   };
+   }
+   ```
+
+   2. Make body class of custom widget(sample-widget-impl.h).
+
+   ```
+   #include <dali/public-api/adaptor-framework/widget-impl.h>
+
+   // HANDLE INCLUDES
+   #include "sample-widget.h"
+
+   namespace Demo
+   {
+   namespace Internal
+   {
+   class SampleWidget : public Dali::Internal::Adaptor::Widget
+   {
+   public:
+     SampleWidget();
+     ~SampleWidget();
+     static Demo::SampleWidget New();
+     virtual void OnCreate( const std::string& contentInfo, Dali::Window window );
+     virtual void OnTerminate( const std::string& contentInfo, Dali::Widget::Termination type );
+     virtual void OnPause();
+     virtual void OnResume();
+     virtual void OnResize( Dali::Window window );
+     virtual void OnUpdate( const std::string& contentInfo, int force );
+
+   protected:
+     // Undefined
+     SampleWidget(const SampleWidget&);
+     SampleWidget& operator=(SampleWidget&);
+   };
+   } // namespace Internal
+   } // namespace Dali
+   ```
+
+   3. Implement each lifecycle functions.
```

Comment:
Implement each lifecycle functions.

Add colon at the end of the sentence:
Implement each lifecycle functions: